### PR TITLE
Sahara mitaka part 2

### PR DESCRIPTION
**Pull request by jeremyfreudberg**

### CODE REVIEW

**Comment by radonm** [comment]
+1

File: sahara/manifests/init.pp [hunk 1]
```diff
@@ -156,6 +156,7 @@
     ensure => $package_ensure,
     name   => 'openstack-sahara-ui',
     tag    => 'openstack',
+    notify  => Service['httpd'], # only restarts if change
```
Comment (kamfonik) [inline sahara/manifests/init.pp]: I'll be nitpicky and point out you need to move all the `=>` here for proper style.

**Review by kamfonik — APPROVED** [review]
+1

File: sahara/manifests/init.pp (hunk 1 above)
Comment (kamfonik) [reply sahara/manifests/init.pp]: actually I think you just have an extra space, maybe

Comment (jeremyfreudberg) [reply sahara/manifests/init.pp]: done

**Comment by radonm** [comment]
Looks better now...

**Pull request merged**